website/content/userguide/_index.markdown

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Userguide"
+title: "Configuration"
```

Comment:
The rest of this file still mentions “Userguide”. That should probably be updated to “Configuration” as well.